horovod/mxnet/__init__.py

```diff
@@ -68,6 +69,22 @@ def set_wd_mult(self, args_wd_mult):
         self._optimizer.set_wd_mult(args_wd_mult)
 
 
+class DistributedTrainer(mx.gluon.Trainer):
+    def __init__(self, params, optimizer, optimizer_params=None):
```

Comment:
It is worthwhile to document the difference between `DistributedTrainer` and `Trainer`, that `DistributedTrainer` performs allreduce(summation) and average, while `Trainer` only performs allreduce(summation)? That will help existing Trainer + kvstore users understand how gradient is calculated

Reply:
Added